gtfs/spec/en/reference.md

```diff
@@ -77,105 +97,104 @@ The following example demonstrates how a field value would appear in a comma-del
+|   |  | | * **0** or blank - Stop. A location where passengers board or disembark from a transit vehicle. | 
+|   |  | | * **1** - Station. A physical structure or area that contains one or more stop. |  
+|   |  | | * **2** - Station Entrance/Exit. A location where passengers can enter or exit a station from the street. The stop entry must also specify a parent_station value referencing the stop ID of the parent station for the entrance. | 
+|  parent_station | Enum | Optional | For stops that are physically located inside stations, the **parent_station** field identifies the station associated with the stop. To use this field, stops.txt must also contain a row where this stop ID is assigned location type=1. |  
+|   |  | | • If this stop ID represents **a stop located inside a station**, this entry's location type must be **0 or blank**, and this entry's parent_station field contains **the stop ID of the station where this stop is located and the stop referenced by parent_station must have location_type=1.** |
+|   |  | | • If this stop ID represents **a stop located outside a station**, this entry's location type must be **0 or blank**, and this entry's parent_station field contains **a blank value (the parent_station field doesn't apply to this stop).** |
+|   |  | | • If this stop ID represents **a station**, this entry's location type must be **1**, and this entry's parent_station field contains **a blank value (stations can't contain other stations).** |
+|  stop_timezone | Timezone | Optional | The **stop_timezone** field contains the timezone in which this stop, station, or station entrance is located. If omitted, the stop should be assumed to be located in the timezone specified by **agency_timezone** in [agency.txt](#agencytxt).   When a stop has a parent station, the stop is considered to be in the timezone specified by the parent station's **stop_timezone** value. If the parent has no stop_timezone value, the stops that belong to that station are assumed to be in the timezone specified by **agency_timezone**, even if the stops have their own **stop_timezone** values. In other words, if a given stop has a **parent_station** value, any **stop_timezone** value specified for that stop must be ignored.  Even if **stop_timezone** values are provided in stops.txt, the times in [stop_times.txt](#stop_timestxt) should continue to be specified as time since midnight in the timezone specified by **agency_timezone** in agency.txt. This ensures that the time values in a trip always increase over the course of a trip, regardless of which timezones the trip crosses. |
```

Comment:
`git blame` lead me here, I have a question.

The `stop_timezone` definition says that

> [...] if `stop_timezone` values are provided in `stops.txt`, the times in `stop_times.txt` should continue to be specified **as time since midnight** in the timezone specified by `agency_timezone` in `agency.txt`.

But the newly added GTFS Time definition [says](https://github.com/google/transit/pull/104/files#diff-3ecf0760eb54b4953728042a1e30586705dc2335807be94faae0de5829cd12a1R51):

> The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur).

I assume that the latter is the correct definition, and that the explanation for `stop_timezone` should be adapted?